### DISCUSSION TITLE
How to try tauri to develop Android app?

### DISCUSSION THREAD

**Discussion by cxgreat2014** [tauri-apps/tauri#3845] [General]
I can't wait to try tauri to develop Android App, is there any place to try/check the current work progress?

**Comment by FabianLars** [comment]
tauri itself doesn't have any android related stuff implemented yet as we're focused on v1 stable. Therefore we also don't have any kind of tracking issues or whatever.

The current implementation is only in [wry](https://github.com/tauri-apps/wry), see [here](https://hackmd.io/XIcEwk4GSxy8APZhSa0UnA?view) for notes on how to test it. Currently the best way to stay up to date is to join our discord and follow the #tao-wry channel.
